ui_files/ui_optimization_fluence_params.ui

```diff
@@ -326,6 +326,9 @@
                <property name="text">
                 <string/>
                </property>
+               <property name="checked">
+                <bool>true</bool>
```

Comment:
Is this a requested change? I can't find any context (e.g. GitHub issue or commit message) for it.

Reply:
It a valid change / [property ](https://doc.qt.io/qt-5/qcheckbox.html#tristate-prop)that comes from Qt Designer. I don't know why it appears here.